Web/Redirection_Based_Privilege_Escalation.md

```diff
@@ -0,0 +1,33 @@
+/*
+Title: Redirection Based Privilege Escalation
```

Comment:
So this is XSS, and sorta like XSRF but local? Got an example of one? I'm sorry I haven't heard of this before.